android/src/main/kotlin/io/chirp/chirpsdk/ChirpsdkPlugin.kt

```diff
@@ -11,14 +11,15 @@ import io.flutter.plugin.common.MethodChannel.MethodCallHandler
 import io.flutter.plugin.common.MethodChannel.Result
 import io.flutter.plugin.common.PluginRegistry.Registrar
 
-import io.chirp.connect.ChirpConnect;
-import io.chirp.connect.interfaces.ConnectEventListener;
-import io.chirp.connect.models.ChirpConnectState;
-import io.chirp.connect.models.ChirpError
+import io.chirp.chirpsdk.interfaces.ChirpEventListener
+import io.chirp.chirpsdk.models.ChirpSDKState
+import io.chirp.chirpsdk.models.ChirpError
 
 
 class ChirpsdkPlugin(val activity: Activity) : MethodCallHandler {
 
+  //TODO: Don't see the reason why we need multiple StreamHandlers for the same ChirpEventListener events.
```

Comment:
We may be able to reduce the amount of `StreamHandlers`, but we will need one for states, one for receiving, and then one for the others I think, because they accept different arguments.